news/803.new.rst

```diff
@@ -0,0 +1 @@
+Add a hook for ``toga``, which has metadata.
```

Comment:
All packages *have* metadata. It's really about whether a package feels the need to loop it up at runtime.

Reply:
Reckon we should add couple of more examples to complement https://github.com/pyinstaller/pyinstaller-hooks-contrib/blob/d40669c89794e7563c3d0b1f9bace7b353cd58ce/news/README.txt#L38